phpBB/includes/functions.php

```diff
@@ -4138,7 +4149,13 @@ function page_header($page_title = '', $display_online_list = false, $item_id =
 		'RECORD_USERS'					=> $l_online_record,
 
 		'PRIVATE_MESSAGE_COUNT'			=> (!empty($user->data['user_unread_privmsg'])) ? $user->data['user_unread_privmsg'] : 0,
-		'CURRENT_USER_AVATAR'			=> phpbb_get_user_avatar($user->data),
+		'CURRENT_USER_AVATAR'			=> $avatar['html'],
+		'CURRENT_USER_AVATAR_LAZY'		=> $avatar['lazy'],
+		'CURRENT_USER_AVATAR_SOURCE'	=> $avatar['src'],
+		'CURRENT_USER_AVATAR_TITLE'		=> $avatar['title'],
+		'CURRENT_USER_AVATAR_TYPE'		=> $avatar['type'],
+		'CURRENT_USER_AVATAR_WIDTH'		=> $avatar['width'],
```

Comment:
I do not think we really need width and height? Here width and height is more for upload restrictions not for display. The dimensions will be barcodes by the classes for all use cases